tensorflow/python/eager/profiler_test.py

```diff
@@ -27,6 +27,7 @@
 from tensorflow.python.framework import constant_op
 from tensorflow.python.framework import test_util
 from tensorflow.python.platform import gfile
+from tensorflow.python.platform import test
```

Comment:
So finally found out the cause of the test failure. This import conflicts with `from tensorflow.python.eager import test` which is why we are getting the test failure. Could you please change the condition on line 51 to use test_util.IsBuiltWithROCm or else alias this import?

Reply:
good catch.

pushed out a new commit to use `test_util.isBuiltWithROCm` instead. 

please re-approve.

thanks